testutils/vcamera/vcamera.go

```diff
@@ -154,25 +149,21 @@ func startStream(config *Config, dev device) (<-chan struct{}, error) {
 		return nil, err
 	}
 
-	// wait for cmd to finish to avoid leak
-	config.activeBackgroundWorkers.Add(1)
-	go func() {
-		defer config.activeBackgroundWorkers.Done()
+	// Wait for cmd to finish to avoid leak.
+	config.workers.AddWorkers(func(ctx context.Context) {
```

Comment:
A note to reviewers: neither the old version nor the new version directly watches the context. However, line 134 starts the command with the context from the StoppableWorkers, so _that's_ where things should cancel early if we call `config.workers.Stop()`.